student-work/2023/Deborah-Sorrels/readme.md

```diff
@@ -1 +1,110 @@
+# Deborah Sorrels  
+#### 054-599-6961 | Deborah.Sorrels@topper.wku.edu  
 
+## Work Experience  
+
+**August 2019-September 2020: Transcriptionist, TAB Services**, Modiin, Israel   
+* Transcribing verbatim documents of court cases from the New York City Department of 
+Education  
+
+**2015-2019: Freelance Editor, Writer**, Various organizations   
+* Edited a grant proposal for the Head of Anesthesiology at RAMBAM Hospital   
+* Edited two grant proposal for Tiltan Signals   
+* Edited English sections of a high-tech website in America   
+* Created marketing advertisement for Mon Salon in Israel   
+* Edited and advised students from International School at Haifa University on final graduation 
+projects  
+
+**2015: Intern, Givat Haviva**, Hadera, Israel   
+* Worked as an intern during the time that they were preparing for the 3rd Annual Givat Haviva 
+Conference which launched their "Joint Vision" program that promoted their ideas for the 
+creation for a shared society between Jews and Arabs in Israel   
+* Contacted, via email, Jewish and Arab organizations in North America to try to get participants, 
+contacted multiple Jewish and Arab organizations and all members of the Israeli Knesset for 
+participation, contacted multiple local foreign embassies to try and get participants   
+* Was successful in getting a couple of ambassadors and Knesset members to attend   
+* Subsequently, published an article for Givat Haviva about the conference, see below  
+
+**2011-2013: Graduate Teaching Assistant**, Western Kentucky University     
+* Co-taught undergraduate level courses in the Hebrew Bible and Introduction to Religious 
+Studies   
+* Was responsible for grading all tests and assignments   
+* Ran tutoring sessions for students that were interested prior to tests  
+
+**2007-2011: Teacher**, Board of Education, Warren and Butler Counties, USA   
+* Taught classes at high school level   
+* Taught classes in various fields ranging from English as a Second Language, Art, Special 
+Education, Language Arts, Science, Math, and Computer Science  
+
+**1994-2001: Manager of Course and Training for New Employees at Toyota**, SEWS, Inc, 
+Kentucky   
+* Trained all new hires by working with them in a one-on-one environment teaching them about 
+the specifics of their particular positions in the company which included such elements as 
+individual responsibilities and duties, as well as, how to fill out and file required for the various 
+positions  
+
+## Academic and Professional Education  
+
+**2020: Yandex Data Analysis Coding Bootcamp**    
+**Completed Projects Using Python, SQL, and Pandas**  
```

Comment:
if these projects are hosted anywhere and are in the public domain - provide a link. This is something to be proud of. 
If they are not, consider adding them to your GitHub repository